### PR TITLE
Local Hospital Capacity + tests!

### PR DESCRIPTION
**Pull request by rofranklin**
well, here ya go

### CODE REVIEW

**Comment by benjamin-musil** [comment]
I'm Ben and I approve this pull request.

File: tests/test_app.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    def setUpClass(cls):
+        app.app.testing = True
+        cls.client = app.app.test_client()
+
+
+class TestGetHospitalCapacity(TestApp):
+    route = '/local-hospital-capacity'
+
+    @mock.patch('app.requests.get')
+    def test_get_hospital_capacity__raises_http_error(self, mock_get):
```
Comment (mdyl) [inline tests/test_app.py:19]: Is the double _ a convention for tests? Just curious, I don't think I've ever seen it before.

File: app.py [hunk 2]
```diff
@@ -1,12 +1,29 @@
...
 def hello_world():
     return 'Hello World!'
 
 
+@app.route('/local-hospital-capacity', methods=['GET'])
+def get_cap():
+    responses = requests.get('https://covid19-server.chrismichael.now.sh/api/v1/AggregatedFacilityCapacityCounty').json()['data'][0]['table']
+    tx_response = []
+    for response in responses:
+        if response['State'] == 'TX' and response['CountyName'] == 'Bexar':
```
Comment (mdyl) [inline app.py:21]: Are we planning on changing this later to not be hard coded to Bexar and TX?

Comment (nwiatrek) [reply app.py:21]: yeah i think it should be pretty easy to add in some querying bits in the url, its not much more code than whats already been written

**Pull request merged**